### PR TITLE
mgr/status: Fix 'fs status' json output

### PR DESCRIPTION
**Pull request by kotreshhr**
Fix rank field of standby-replay mds daemon in 'ceph fs status'
    json output. The rank field for standby-replay mds daemons
    were incorrect. It should be of the format '{rank}-s',
    where {rank} is the active mds which the standby-replay is
    following.
    
    Fixes: https://tracker.ceph.com/issues/67978
    Signed-off-by: Kotresh HR <khiremat@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [x] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: src/pybind/mgr/status/module.py [hunk 1]
```diff
@@ -161,7 +161,7 @@ def handle_fs_status(self,
 
                 if output_format in ('json', 'json-pretty'):
                     json_output['mdsmap'].append({
-                        'rank': rank,
+                        'rank': "{0}-s".format(daemon_info['rank']),
```
Comment (dparmar18) [inline src/pybind/mgr/status/module.py:164]: maybe use f-string?

Comment (kotreshhr) [reply src/pybind/mgr/status/module.py]: Done

**Review by dparmar18 — APPROVED** [review]

**Comment by kotreshhr** [comment]
jenkins test api

**Comment by kotreshhr** [comment]
Sorry, closed this PR by accident. Reopend it.

**Review by joscollin — APPROVED** [review]

**Comment by vshankar** [comment]
This PR is under test in [https://tracker.ceph.com/issues/68092](https://tracker.ceph.com/issues/68092).

**Review by vshankar — APPROVED** [review]
https://tracker.ceph.com/projects/cephfs/wiki/Main#wip-vshankar-testing-20240917043942-debug

**Pull request merged**